pkg/sink/elasticsearch.go

```diff
@@ -172,7 +172,7 @@ func (e *Elasticsearch) flushIndex(ctx context.Context, indexCfg config.ELSIndex
 }
 
 // SendEvent sends event notification to Elasticsearch
-func (e *Elasticsearch) SendEvent(ctx context.Context, event events.Event) (err error) {
+func (e *Elasticsearch) SendEvent(ctx context.Context, event events.Event, _ []string) (err error) {
```

Comment:
Shouldn't Elasticsearch also use the `eventSources`? For outgoing webhook it makes sense to not use it, but Elasticsearch supports multiple indices, so... 🤔

Reply:
Elasticsearch source bindings now added.